resources/i18n/fr.json

```diff
@@ -195,7 +195,7 @@
                 "maxBitRate": "Bitrate maximum",
                 "updatedAt": "Mis à jour le",
                 "createdAt": "Créé le",
-                "downloadable": "Autoriser les téléchargements?"
+                "downloadable": "Autoriser les téléchargements ?"
```

Comment:
In French (as in "from France") typography, a question mark should be preceded by a non-breaking space. [Source](https://fr.wikipedia.org/wiki/Point_d'interrogation#Usage).

> En [France](https://fr.wikipedia.org/wiki/France), il est d'usage de placer une [espace fine insécable](https://fr.wikipedia.org/wiki/Espace_fine_ins%C3%A9cable)[1](https://fr.wikipedia.org/wiki/Point_d'interrogation#cite_note-1) avant le point d'interrogation, ou une [espace insécable](https://fr.wikipedia.org/wiki/Espace_ins%C3%A9cable) si la fine n'est pas disponible[2](https://fr.wikipedia.org/wiki/Point_d'interrogation#cite_note-2).
This rule also applies to all punctuation marks made of two symbols: `!:;` (I have fixed a few of them as well).

One can argue that this is no longer totally true in other French-speaking countries but since this file is already making use of spaces before punctuation marks in some places, I think it makes sense to make things consistent.